### PR TITLE
[Chore] Replace Cocoapods-keys with Arkana

### PR DESCRIPTION
**Pull request by ducbm051291**
- #70 

## What happened

- We need to replace Cocoapods-keys with Arkana because Cocoapods-keys has some issues somehow it can't retrieve the keys values and it is old
 
## Insight

- Add .arkana.yml file to configurate Arkana
 - Update .env with ClientId and ClientSecret
- Update .gitignore to ignore Arkana generated folder
- Add Arkana in gemfile
- Add Arkana Generated Pods in Podfile

```
Step 1: Run `bundle install` to install Arkana
Step 2: Run `bundle exec arkana` to generate Pods
Step 3: Run `bundle exec pod installl` to install generated Pods
```
 
## Proof Of Work

Screenshot 2022-11-29 at 16 53 51

Screenshot 2022-11-29 at 16 54 23

Screenshot 2022-11-29 at 16 54 06

### CODE REVIEW

File: .github/workflows/deploy_Firebase.yml [hunk 1]
```diff
@@ -53,6 +53,9 @@ jobs:
       # if: steps.bundleCache.outputs.cache-hit != 'true'
       run: bundle install
 
+    - name: Run Arkana
```
Comment (blyscuit) [inline .github/workflows/deploy_Firebase.yml:56]: You will need a script to read value from Secrets and populate to `.env` or else the build wouldn't work correctly. You can see in [this section](https://docs.github.com/en/actions/security-guides/encrypted-secrets#storing-base64-binary-blobs-as-secrets) for example, then write the decoded string to .env file like [this](https://github.com/suho/kmm-ic/blob/develop/.github/workflows/deploy_ios_staging_build.yml#L49).

File: .github/workflows/deploy_Release_Firebase.yml [hunk 2]
```diff
@@ -46,6 +46,9 @@ jobs:
             
     - name: Bundle install
       run: bundle install
+      
+    - name: Run Arkana
```
Comment (blyscuit) [inline .github/workflows/deploy_Release_Firebase.yml:50]: Same.

**Review by blyscuit — APPROVED** [review]

File: .env.example [hunk 3]
```diff
@@ -0,0 +1,4 @@
+ClientIdDebug = "SET YOUR VALUE HERE"
```
Comment (Shayokh144) [inline .env.example:1]: just a question, when do you set the values here?

**Review by Shayokh144 — APPROVED** [review]

File: .env.example (hunk 3 above)
Comment (ducbm051291) [reply .env.example:1]: @Shayokh144 on your local machine :D we won't commit this file 😀

Comment (Shayokh144) [reply .env.example:1]: thanks .. that was helpful 👍

**Pull request merged**